Client/game_sa/CGameSA.h

```diff
@@ -262,6 +262,8 @@ class CGameSA : public CGame
     bool IsVehicleBurnExplosionsEnabled() const noexcept override { return m_isVehicleBurnExplosionsEnabled; }
     void SetVehicleBurnExplosionsEnabled(bool isEnabled) override;
 
+    bool IsVehicleEngineManualModeEnabled() const noexcept override { return *(unsigned char*)0x64BC03 == 0x90; }
```

Comment:
This method should be in `Multiplayer SA` as well. Because it's logically connected to memory manipulations inside `SetVehicleEngineManualModeEnabled`.

Reply:
See https://wiki.multitheftauto.com/wiki/Coding_info
Game SA is not the right place for low level modifications. And simplicity isn't the main principle when it takes to code organization questions. There must be also coherency and predictability.

Reply:
Those modification are obviously erroneous. So please, follow the instructions.

Reply:
I want to move setter to Game SA like in my original implementation. I think this is the right way to do it because hook that is used to modify aircraft logic is in Game SA (CVehicleSA.cpp).

Reply:
> Game SA isn't intended to be used for the modifications

By the way, it means that most if not all hooks from CVehicleSA.cpp should be moved to Multiplayer SA. And it kinda doesn't make any sense. It's all VehicleSA related code. Including the feature of this PR.

Reply:
I can understand your position. But I prefer adhere to guidlines.
And there's another link for your information: https://github.com/multitheftauto/mtasa-blue/wiki/Dev-Tips

Reply:
Then I guess we should close this PR and someone else can use the code to reimplement this feature. I don't want to be the author if it's implemented in proposed form.

Regarding guidelines - it doesn't forbid using Game SA for low level modifications. It's actually mentioned there that it is used for low level modifications of game element related entities (which this feature relies on).

The fact that multiple similar PRs were merged without issues recently makes me wonder why this one resulted in a problem. In any case, it's up to you to either close this PR or keep it open.

Reply:
I'm not going to close this PR. The words above are just my opinion. Hopefully there are other developers with write access and maybe they will take a different view.

Reply:
> Ok. I will keep this conversation not resolved for more feedback.
> 
> Also I did check if it's easy or not to directly include MultiplayerSA into CVehicleSA.cpp - it's not, there are conflicts like I previously guessed.

What do you mean by including MultiplayerSA? Isn't it enough to just `const CMultiplayer* const multiplayer = g_pCore->GetMultiplayer();`? You can extend CMultiplayer interface if needed.